Gemfile

```diff
@@ -9,6 +9,7 @@ group :development, :test do
   gem 'hashie'
   gem 'rake'
   gem 'rubocop', '0.51.0'
+  gem 'virtus'
```

Comment:
This can't be used at all in the final PR.

Reply:
There are a few tests which use Virtus to verify that it can be used as a custom type, it means I need to remove the :thinking:

Reply:
I worry that we'll leave something that works in tests but not in prod. Maybe those can become integration tests with their own Gemfile?

Reply:
You are right, Virtus isn't needed anymore. I removed it and adjusted tests testing custom types :slightly_smiling_face: